settings.py

```diff
@@ -721,7 +721,7 @@ def get_env_list(name):
 TEST_RUNNER = 'django.test.runner.DiscoverRunner'
 
 # Email server
-EMAIL_USE_TLS = True
+EMAIL_USE_TLS = os.environ.get('WEBLATE_EMAIL_USE_TLS', 'True') == 'True'
```

Comment:
We test other boolean variables against '1' and '0', so I think it's better to stay consistent...